### PR TITLE
Tutorial Updates (Pre-Phase 2) *WIP*

### PR DESCRIPTION
**Pull request by nlarew**
## Jira

- [x] (DOCSP-11232): Unite Realm Web and Mobile Schema to Sync same data between the two Tutorials
- [x] (DOCSP-11200): [Realm Web SDK] Indicate that Sync is not supported
- [x] (DOCSP-11468): Split Rules/Sync into Client-Specific Tutorials
- [ ] (DOCSP-11223): Update Realm Web tutorial to use _id on the assignee relationship
- [ ] (DOCSP-11473): Update iOS tutorial to use string User._id
- [ ] (DOCSP-11474): Update Android tutorial to use string User._id
- [ ] (DOCSP-11475): Update RN tutorial to use string User._id
- [ ] (DOCSP-11476): Update Node tutorial to use string User._id

### Unite the Schemas

Use consistent schemas for all task tracker tutorials. `User._id` is now the user's string id value (was ObjectId), the relationship now always points from `Task.assignee (string) -> User._id (string)`, and `_partition` is no longer required.

- [x] [Set Up the Task Tracker Tutorial Backend](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/tut-relationship/tutorial/realm-app.html)

### Split Rules/Sync Procedures

Move the rules/sync procedures into individual client tutorials

- [x] [Web](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/tut-relationship/tutorial/web-graphql.html#a-define-data-access-permissions)
- [x] [Node](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/tut-relationship/tutorial/nodejs-cli.html#a-define-data-access-permissions-and-enable-sync)
- [x] [React Native](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/tut-relationship/tutorial/react-native.html#define-data-access-permissions-and-enable-sync)
- [x] [Android](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/tut-relationship/tutorial/android-kotlin.html#define-data-access-permissions-and-enable-sync)
- [x] [iOS](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/tut-relationship/tutorial/ios-swift.html#define-data-access-permissions-and-enable-sync)

### Tutorial Code Updates

Update individual client code in [mongodb-university/realm-tutorial](https://github.com/mongodb-university/realm-tutorial) to work with the new schema

- [ ] [Web]() **TODO**
- [ ] [Node]() **TODO**
- [ ] [React Native]() **TODO**
- [ ] [Android]() **TODO**
- [ ] [iOS]() **TODO**

### Tutorial Text Updates

Update the client tutorials to match the updated code

- [ ] [Web]() **TODO**
- [ ] [Node]() **TODO**
- [ ] [React Native]() **TODO**
- [ ] [Android]() **TODO**
- [ ] [iOS]() **TODO**

### CODE REVIEW

File: source/tutorial/web-graphql.txt [hunk 1]
```diff
@@ -71,7 +71,49 @@ Before you get started, you'll need the following:
...
+   update the ``/src/realm/RealmApp.tsx`` file with your App ID, which you can
+   find in the {+ui+}.
+
+A. Define Data Access Permissions
+---------------------------------
+
+.. admonition:: GraphQL Client Only
+   :class: important
+
+   These permissions only apply if you do not intend to connect any non-web
```
Comment (ianpward) [inline source/tutorial/web-graphql.txt:91]: is this true? I thought permissions still applied they were limited to the more limited sync permission rules where a partition is read or read/write per iuser

File: source/tutorial/web-graphql.txt [hunk 2]
```diff
@@ -38,22 +46,14 @@ The app is a task tracker that allows users to:
...
-
-.. admonition:: Download the Complete Source Code
-   :class: note
-   
-   We host this tutorial application's :github:`complete and ready-to-run
-   source code <mongodb-university/realm-tutorial/tree/main/web>` on GitHub.
-   Just follow the instructions in ``README.md`` to get started. Don't forget to
-   update the ``/src/realm/RealmApp.tsx`` file with your App ID, which you can
-   find in the {+ui+}.
+   The Realm Web SDK does not support :ref:`sync <sync-overview>`, so the app
```
Comment (ianpward) [inline source/tutorial/web-graphql.txt:52]: couldn't we make it semi real time by continually re-querying the backend? the inventory demo app the field team uses does this by checkng an autorefresh box. Do you want to share the code the field team uses in their demo app?

File: source/tutorial/web-graphql.txt (hunk 1 above)
Comment (nlarew) [reply source/tutorial/web-graphql.txt:91]: Yeah correct. The "these" here refers to non-sync permissions. I'll update to clarify.

File: source/tutorial/web-graphql.txt (hunk 2 above)
Comment (nlarew) [reply source/tutorial/web-graphql.txt:52]: We could probably add polling but for now we don't support it. It looks pretty simple in Apollo though, so I'll look into adding it https://www.apollographql.com/docs/react/data/queries/#polling

Comment (nlarew) [reply source/tutorial/web-graphql.txt:52]: https://jira.mongodb.org/browse/DOCS-13798

**Pull request merged**